api/src/internal/oscc.h

```diff
@@ -7,9 +7,32 @@
  #ifndef _OSCC_INTERNAL_H
  #define _OSCC_INTERNAL_H
 
+#include <net/if.h>
+#include <stdbool.h>
 
 #define CONSTRAIN(amt, low, high) ((amt) < (low) ? (low) : ((amt) > (high) ? (high) : (amt)))
 
+struct can_contains {
+    bool is_oscc;
+    bool has_vehicle;
+}can_contains_s;
+
+struct vehicle_can_desc {
+    bool has_steering_angle;
+    bool has_brake_pressure;
+    bool has_wheel_speed;
+}vehicle_can_desc_s;
+
+struct oscc_can_desc {
+    bool has_torqe_report;
```

Comment:
I can't tell if this is a typo or if the struct member names are intentionally the same size, but the "u" was omitted from this variable.

Also, is this a throttle report? Torque can apply to both steering and throttle and it's got me a bit confused.